pkg/querier/tenantfederation/merge_queryable.go

```diff
@@ -143,7 +149,8 @@ func (m *mergeQuerier) LabelValues(name string, matchers ...*labels.Matcher) ([]
 		name = m.idLabelName
 	}
 
-	return m.mergeDistinctStringSlice(func(ctx context.Context, q storage.Querier) ([]string, storage.Warnings, error) {
+	return m.mergeDistinctStringSlice(ctx, func(ctx context.Context,
+		q storage.Querier) ([]string, storage.Warnings, error) {
```

Comment:
[nit] Please rollback this change.

Reply:
you are right, this context is not used in callback function so it's not necessary to pass it to this method. done